h2o-docs/src/product/cloud-integration/ec2-and-s3.rst

```diff
@@ -168,39 +168,17 @@ Minio Instance
 
 Minio Cloud Storage is an alternative to Amazon AWS S3. When using a Minio server, the following additional parameters are specified in the Java launch command:
 
-- ``endpoint``: Specifies a minio server instance (including address and port). This overrides the existing endpoint, which is currently hardcoded to be AWS S3.
+- ``endpoint``: Specifies a Minio server instance (including address and port). This overrides the existing endpoint, which is currently hardcoded to be AWS S3.
 
 - ``enable.path.style``: Specifies to override the default S3 behavior to expose every bucket as a full DNS enabled path. Note that this is a Minio recommendation.
 
-1. Edit the properties in the ``core-site.xml`` file to include your these new parameters as well as the Access Key ID and Access Key. Refer to the following example:
+1. Launch H2O by entering the following in the command line:
 
   ::
 
-      <property>
-        <name>Dsys.ai.h2o.persist.s3.endPoint</name>
-        <value>example.minio.io:9000</value>
-      </property>
-      <property>
-        <name>Dsys.ai.h2o.persist.s3.enable.path.style</name>
-        <value>true</value>
-      </property>
-      <property>
-        <name>Daws.AccessKeyId</name>
-        <value>[MINIO SECRET KEY]</value>
-      </property>
-
-      <property>
-        <name>Daws.SecretAccessKey</name>
-        <value>[MINIO SECRET ACCESS KEY]</value>
-      </property>
-
-2. Launch with the configuration file ``core-site.xml`` by entering the following in the command line:
+      java -Dsys.ai.h2o.persist.s3.endPoint=s3.amazonaws.com -jar h2o.jar
```

Comment:
I think this should change to `java -Dsys.ai.h2o.persist.s3.endPoint=https://play.min.io:9000 -Dsys.ai.h2o.persist.s3.enable.path.style=true -jar h2o.jar`, both parameters are needed for Minio to work in my testing.